lightning/backend/tidb.go

```diff
@@ -366,7 +366,7 @@ func (be *tidbBackend) FetchRemoteTableModels(schemaName string) (tables []*mode
 			SELECT table_name, column_name
 			FROM information_schema.columns
 			WHERE table_schema = ?
-			ORDER BY table_name;
+			ORDER BY table_name, ordinal_position;
```

Comment:
Could you add a test?

Reply:
It's hard to add a test for it. The current test framework is based on `sqlmock`. The `sqlmock` can't support to test this case. There is one way to implement this test: import the whole tide